src/m365/spo/commands/tenant/tenant-recyclebinitem-remove.spec.ts

```diff
@@ -21,16 +21,16 @@ describe(commands.TENANT_RECYCLEBINITEM_REMOVE, () => {
   let commandInfo: CommandInfo;
 
   before(() => {
-    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
-    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
-    sinon.stub(pid, 'getProcessName').callsFake(() => '');
-    sinon.stub(session, 'getId').callsFake(() => '');
-    sinon.stub(spo, 'getRequestDigest').callsFake(() => Promise.resolve({
+    sinon.stub(auth, 'restoreAuth').resolves();
+    sinon.stub(telemetry, 'trackEvent').returns();
+    sinon.stub(pid, 'getProcessName').returns('');
+    sinon.stub(session, 'getId').returns('');
+    sinon.stub(spo, 'getRequestDigest').resolves({
```

Comment:
It turns out there are too many stubs in this file. Most are unnecessary. We can change this reference and remove all others:

```suggestion
    sinon.stub(spo, 'ensureRequestDigest').resolves({
```